ios/yaysornays.xcodeproj/project.pbxproj

```diff
@@ -13,7 +13,7 @@
 		13B07FBF1A68108700A75B9A /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		13B07FC11A68108700A75B9A /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		49C9263CA9D38372F7C649FB /* libPods-yaysornays-yaysornaysTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 261794707BF62557E927F575 /* libPods-yaysornays-yaysornaysTests.a */; };
-		81AB9BB82411601600AC10FF /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */; };
+		9A7CA252262503C500ADD141 /* Launch Screen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 9A7CA251262503C500ADD141 /* Launch Screen.storyboard */; };
```

Comment:
Is this space intentional?

Reply:
I believe this was changed by the bundler itself after I manipulated the file in xCode. So in short, this is some behind the scenes magic that xCode does because I did not change this file manually.